### PR TITLE
Update for readonly_selection element

### PR DESCRIPTION
**Pull request by mchf**

### CODE REVIEW

**Comment by imobachgs** [comment]
Is `master` the right target branch?

File: package/yast2-installation-control.changes [hunk 1]
```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Wed Dec 14 08:40:48 UTC 2016 - mfilka@suse.com
+
+- Added new "readonly_selection" parameter for CASP workflow. It
+  makes software proposal readonly
```
Comment (kobliha) [inline package/yast2-installation-control.changes]: It makes `software proposal` or rather `a particular proposal item` read only?

Comment (mvidner) [inline package/yast2-installation-control.changes]: FATE reference missing: https://fate.suse.com/321739

**Comment by mchf** [comment]
Solved otherwise in the end.

**Pull request closed without merging**